### PR TITLE
reformat code using clion auto tool

### PR DESCRIPTION
**Pull request by IdanTo1**
Signed-off-by: Idan Tomer <idan.tomer2@gmail.com>

### CODE REVIEW

File: src/Session.cpp [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
 
-Session::~Session()
-{
+Session::~Session() {
     this->clean();
 }
-void Session::split(std::string &actionString, std::vector <std::string>& actionParams, std::string delimiter) {
+
+void
+Session::split(std::string& actionString, std::vector<std::string>& actionParams, std::string delimiter) {
```
Comment (Hroy02) [inline src/Session.cpp]: Seems a little awkward to me

**Review by Hroy02 — DISMISSED** [review]
One Comment, you can merge if you think otherwise because this is by no means an error

File: src/Session.cpp (hunk 1 above)
Comment (IdanTo1) [reply src/Session.cpp]: You're right
It shouldn't happen
I didn't see it

**Review by Hroy02 — APPROVED** [review]
Perfect

**Pull request merged**